auth-web/src/models/business.ts

```diff
@@ -6,6 +6,7 @@ export interface LoginPayload {
     passCode?: string
     phone?: string
     email?: string
+    certifiedByName?: string
```

Comment:
In which case is this not present?

Reply:
I'm not sure in which case this is not present, but having "?" will not break anything if this is not present in any case.

Reply:
It won't break anything but it might be incorrect. This is an anti-pattern.

I suggest you remove the ? and see if anything breaks.

Reply:
It's now removed, and nothing breaks.